reference/7.0/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -554,6 +556,71 @@ B
 A
 ```
 
+If you assign a character range to a string, it's treated the same assigning a
+character array to a string.
```

Comment:
Is it worth noting that specifying non-character strings throws an exception? I verified in a try/catch that it does.

Reply:
I don't think that is necessary.